### PR TITLE
Fix compatibility with Jenkins 2.482+

### PR DESCRIPTION
**Pull request by Vlatombe**
Temporary offline cause is now persisted separately from technical
offline cause. The getter must be used to achieve compatibility.

Fixes #385

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by Vlatombe** [comment]
Seems the tests already fail in `master`, https://ci.jenkins.io/job/Plugins/job/openstack-cloud-plugin/job/master/236/

File: plugin/src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public void setPendingDelete(boolean newVal) {
      */
     public boolean isPendingDelete() {
         // No need  to synchronize reading as offlineCause is volatile
```
Comment (jglick) [inline plugin/src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java:90]:
```suggestion
```

Obsolete?

**Comment by Vlatombe** [comment]
Fixes #385

**Comment by olivergondza** [comment]
@Vlatombe, thanks!

**Pull request merged**